HIP/hip-1056.md

```diff
@@ -848,10 +857,33 @@ message CreateAccountOutput {
 
 ```protobuf
 message EthereumOutput {
+   // Preview block streams used these fields for sidecars and verbose
+   // contract function results
+   reserved 2,3,4;
+
     /**
-     * Result details for an EVM transaction execution
+     * If the associated Ethereum transaction required hydrating its
+     * call data from a file, the Keccak256 hash of the transaction.
+     * <p>
+     * If the Ethereum transaction was inline, this field SHALL NOT be set.
      */
-    EVMTransactionResult evm_transaction_result = 1;
+    bytes ethereum_hash = 1;
+
+    oneof txn_result {
```

Comment:
Q: Why is a `oneof` required?
The input transaction type would signify a Call or Create. 
Even in the case of an `EthereumTransaction` a call or create can be inferred by the presence of `tx.to`